### ISSUE TITLE
Endzone - A World Apart (933820)

### ISSUE THREAD

**Issue by mibli** [ValveSoftware/Proton#4690]
### Compatibility Report
Name of the game with compatibility issues: Endzone: A World Apart
Steam AppID of the game: 933820

### System Information
OS: Manjaro Linux
KERNEL: 5.4.105-1-MANJARO
CPU: AMD Ryzen 5 3600 6-Core
GPU: AMD AMD Radeon RX 5600 XT (NAVI10, DRM 3.35.0, 5.4.105-1-MANJARO, LLVM 11.1.0)
GPU DRIVER: 4.6 Mesa 20.3.4
RAM: 16 GB
Proton version: 5.13-6

### I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x]  that I have checked whether there are updates for my system available

[steam-stdout.log](https://gist.github.com/mibli/7e4b29a0e24326a248fa1d3ab04cabb2#file-steam-933820-log)
[steam-933820.log.tar.gz](https://github.com/mibli/proton-logs/raw/master/steam-933820.log.tar.gz)

### Symptoms
Game hangs up suddently and has to be stopped with steam's "STOP" button.

### Reproduction steps
1. Launch the game
2. Try to play for a few hours (takes from 15min to 2h)

**Comment by kisak-valve** [comment]
Hello @mibli, it looks like you attached a general stdout/stderr log instead of a Proton log. Please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the game hang you're seeing, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

**Comment by mibli** [comment]
> Hello @mibli, it looks like you attached a general stdout/stderr log instead of a Proton log. Please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the game hang you're seeing, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

Thanks for letting me know. I did what You suggested. Here's the [link to the added tar.gz](https://github.com/mibli/proton-logs/raw/master/steam-933820.log.tar.gz)

**Comment by kisak-valve** [comment]
Looks like an access violation (c0000005) happened around 8592.232, which lead to `err:fsync:get_shm Failed to map page 63317 (offset 0xf755000).` and the ```err:virtual:alloc_area_in_reserved_or_between_callback Could not map in reserved area.
part\Endzone.exe: ../../../../proton/wine/dlls/ntdll/unix/virtual.c:1217: create_view: Assertion `!((UINT_PTR)base & page_mask)' failed.``` asserts are fallout after the was game already in a bad state.

You might get lucky running with `PROTON_NO_FSYNC=1 %command%` or `PROTON_NO_FSYNC=1 PROTON_NO_ESYNC=1 %command%` in the game's launch options, but that might only delay the issue or degrade performance.

**Comment by mibli** [comment]
> You might get lucky running with `PROTON_NO_FSYNC=1 %command%` or `PROTON_NO_FSYNC=1 PROTON_NO_ESYNC=1 %command%` in the game's launch options, but that might only delay the issue or degrade performance.

Yep I was just able to play almost 3 hours without crashing with FSYNC disabled, and no meaningful performance drop for me. This is good enough, even if the issue is delayed.

**Comment by JustArchi** [comment]
I realize that the issue is old and may be no longer relevant, but I confirm that `PROTON_NO_FSYNC=1` is still required to avoid crashes in endzone as of March 2022 (1 year later) with latest Proton Experimental, disabling `ESYNC` is not required.

**Comment by kisak-valve** [comment]
Endzone - A World Apart  ( 933820 )

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7983.
@GhChancellor posted on 2024-08-12T08:24:25:

# Compatibility Report
- Name of the game with compatibility issues: Endzone - A World Apart
- Steam AppID of the game: 933820

## System Information
- OS: Ubuntu 24.04 LTS x86_64 
- Kernel: 6.8.0-40-generic 
- CPU: Intel i7-4790K (8) @ 4.400GHz 
- GPU: NVIDIA GeForce RTX 2080 Ti Rev. A
- Driver Version: 4.6.0 NVIDIA 535.183.01 
- Memory: 5154MiB / 15926MiB 
- Proton versions: Proton GE-9.11, Proton experimental, Proton 9.0-2, Proton 8.0-5, Proton 7.0-6, Proton 6.3-8, Proton 5.13-6, Proton 5.0-10
- Log: [steam-933820.zip](https://github.com/user-attachments/files/16579055/steam-933820.zip)

## I confirm:
- [I confirm] that I haven't found an existing compatibility report for this game.
- [I confirm] that I have checked whether there are updates for my system available.



## Symptoms 
The game doesn't start at all. It only "runs" for a few seconds before stopping.

I've tried GE-Proton 9.11, Proton Experimental, Proton 9.0-2, Proton 8.0-5, Proton 7.0-6, Proton 6.3-8, Proton 5.13-6, Proton 5.0-10.

I've also tested various launch options like:
PROTON_NO_ESYNC=1 PROTON_NO_FSYNC=1 %command%

## Reproduction
The game doesn't start at all. It only "runs" for a few seconds (the initial loading screen) before stopping.
